nickhartdev/Overlook
# Login screen

###### What's this PR do?
- fixes login functionality
- adds all necessary spy tests for domUpdates so far
- adds html for login form, user landing page and manager landing page
###### Where should the reviewer start?
domUpdates.js
###### How should this be manually tested?
Run the testing suite. You can also open index.html and mess with different logins to see the different results.
###### Any background context you want to provide?
The updates to the class structure were very fluid and may not all be documented in the tickets here.
###### What are the relevant tickets?
Closes #3
